tests/automated_install.sh

```diff
@@ -64,7 +64,7 @@ done
 
 echo "Checking for GNU screen to watch serial consoles"
 if hash screen 2>/dev/null ; then
-  pushd $(readlink -f $(dirname $0))
+  pushd $(greadlink -f $(dirname $0))
```

Comment:
This looks good; but I would suggest that there be a conditional for the `readlink` binary. Perhaps setting a variable for the `readlink` binary's name? This way the current binary name can be used on Linux and `greadlink` on OS X?